chapter05/drjt/ex3.pl

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/perl
+use 5.010;
+use warnings;
+use strict;
+
+# This is literally the same code, only with default sorts, and a different format for the output. 40 minutes my arse.
+my %total_bytes;
+while ( <> ) {
+  my ( $source, $dest, $bytes ) = split;
+  $total_bytes{$source}{$dest} += $bytes;
+  $total_bytes{$source}{total} += $bytes;
```

Comment:
This running total isn't necessary for this exercise - could have used another of those 40 estimated minutes and save yourself a few operations here :wink: